src/commands/config/generate.js

```diff
@@ -41,6 +42,36 @@ class ConfigGenerateCommand extends Command {
       })
     }
 
+    // Check if --env flag is present
+    if (flags.env) {
+      const generateConfig = {
+        server: {
+          port: parseInt(process.env.LOGCHIMP_SERVER_PORT) || 3000,
+          secretkey: process.env.LOGCHIMP_SECRET_KEY,
+        },
+        database: {
+          host: process.env.LOGCHIMP_PG_HOST,
+          port: parseInt(process.env.LOGCHIMP_PG_PORT) || 5432,
+          user: process.env.LOGCHIMP_PG_USER,
+          password: process.env.LOGCHIMP_PG_PASSWORD,
+          name: process.env.LOGCHIMP_PG_DATABASE,
+          // dotenv returns all environment variables as strings
+          ssl: process.env.LOGCHIMP_PG_SSL ? process.env.LOGCHIMP_PG_SSL === 'true' : true,
```

Comment:
```suggestion
          ssl: process.env.LOGCHIMP_PG_SSL === 'true' ? true : false,
```

Shouldn't it be like this?

Reply:
The default value of `database.ssl` is `true` so I expect we may want the value of `database.ssl` to be `true` when `LOGCHIMP_PG_SSL` is not defined.

Reply:
That's not always the case in case, setting up local SSL for database can be pretty troublesome thing.

I have set it to false even for CI pipeline.

Reply:
A rule of thumb for self-hosted software is to make sure the defaults are as secure as possible. This means that the default value of a value like `database.ssl` should be naturally be `true`.

If a user cannot connect to the database via SSL, or does not want to connect via SSL, they can set the value of the `LOGCHIMP_PG_SSL` environment variable to `false`.

What do you think about this suggestion? Do you think we should set the default to be `true` or `false`?

Reply:
Yeah, you make a good point.